Gordon360/ApiControllers/HousingController.cs

```diff
@@ -52,8 +48,8 @@ public IHttpActionResult GetApartmentInfo()
         /// <returns>The result of submitting the application</returns>
         [HttpPost]
         [Route("save")]
-        [StateYourBusiness(operation = Operation.UPDATE, resource = Resource.HOUSING)]
-        public IHttpActionResult SaveApplication(string username, string [] applicants)
+        //[StateYourBusiness(operation = Operation.UPDATE, resource = Resource.HOUSING)] we need to actually add HOUSING to stateYourBusiness if we do this
```

Comment:
Maybe we should figure out if this needs state your business

Reply:
From what I've read about the "StateYourBusiness", I don't really know what it is or what it does

Reply:
I’ll make a note in the Team schedule to revisit this in a later branch